platform/slz/alz_library_metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/Azure-Landing-Zones-Library/main/schemas/library_metadata.json",
+  "name": "SLZ",
+  "display_name": "Sovereign Landing Zones",
+  "description": "This library provides the reference set of Sovereign Landing Zones (SLZ) policies, archetypes, and management group architecture.",
+  "path": "platform/slz"
```

Comment:
Please add ALZ dependency in here: https://azure.github.io/Azure-Landing-Zones-Library/assets/metadata/#alz-library-dependencies